apps/cms-server/modules/openstad-rte-widget/views/widget.html

```diff
@@ -0,0 +1 @@
+<div data-openstad-rte data-content="{{ data.widget.text.items[0].content }}"></div>
```

Comment:
Hier geldt denk ik hetzelfde; er moet een check omheen of de `data.widget.text.items[0]` niet leeg is.